### PR TITLE
split in reverse

### PR DESCRIPTION
**Pull request by svennergr**
split in reverse

### CODE REVIEW

**Review by matyax — APPROVED** [review]
Amazing reverse reversing!

File: public/app/plugins/datasource/loki/queryUtils.ts [hunk 1]
```diff
@@ -364,15 +372,17 @@ export function combineResponses(currentResult: DataQueryResponse | null, newRes
...
-    combine(currentFrame, newFrame);
+    combine(new MutableDataFrame(currentFrame), newFrame);
   });
 
   return currentResult;
 }
 
-function combine(dest: DataQueryResponseData, source: DataQueryResponseData) {
-  for (let j = 0; j < source.fields[0].values.length; j++) {
+function combine(dest: MutableDataFrame, source: MutableDataFrame) {
```
Comment (ryantxu) [inline public/app/plugins/datasource/loki/queryUtils.ts:381]: if you can avoid `MutableDataFrame` that is better -- it has a LOT of overhead to make things feel simple/easy, but if not required will avoid a bunch

Comment (matyax) [reply public/app/plugins/datasource/loki/queryUtils.ts:381]: Thanks! Removed in https://github.com/grafana/grafana/pull/62767/commits/b1762240bb290661abef255995a27403871fae1d

Comment (svennergr) [reply public/app/plugins/datasource/loki/queryUtils.ts:381]: Thanks for the info @ryantxu!

**Pull request closed without merging**